src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  let result = 0;
```

Comment:
The variable 'result' can be declared as a 'const' since it is reassigned before every return statement. Consider changing 'let' to 'const' to prevent unintentional changes. Refer to checklist item 3.